openjdk-17.yaml

```diff
@@ -120,6 +120,39 @@ subpackages:
       runtime:
         - freetype
         - fontconfig
+  - name: openjdk-11-jre-headless
+    pipeline:
+      - runs: |
+          export _FROM_ROOT=${{targets.destdir}}/usr/lib/jvm/openjdk
+          export _TO_ROOT=${{targets.subpkgdir}}/usr/lib/jvm/openjdk-jre
+          mkdir -p "$_TO_ROOT"
+          mv "$_FROM_ROOT"/lib "$_TO_ROOT"
+
+          # move back ct.sym
+          mkdir -p "$_FROM_ROOT"/lib
+          mv "$_TO_ROOT"/lib/ct.sym "$_FROM_ROOT"/lib/
+          
+          mkdir -p "$_TO_ROOT"/bin
+
+          for i in java \
+                  jfr \
+                  jrunscript \
+                  keytool \
+                  rmiregistry; do
+                  mv "$_FROM_ROOT/bin/$i" "$_TO_ROOT/bin/$i"
+          done
+
+          mv "$_FROM_ROOT/legal"      "$_TO_ROOT"
+          mv "$_FROM_ROOT/conf"       "$_TO_ROOT"
+          mv "$_FROM_ROOT/release"    "$_TO_ROOT"
+          cp "./ASSEMBLY_EXCEPTION"   "$_TO_ROOT"
+          cp "./LICENSE"              "$_TO_ROOT"
+          cp "./README.md"            "$_TO_ROOT"
+    description: "Oracle OpenJDK 17 (JRE headless)"
```

Comment:
Could you remove "Oracle" to stay consistent with the other package descriptions?